tests/Avalonia.Markup.UnitTests/Extensions/IEnummerableExtension.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections;
+
+namespace Avalonia.Markup.UnitTests
+{
+    static class IEnummerableExtension
+    {
+        public static object ElementAt(this IEnumerable source, int index)
+        {
+            var i = -1;
+            var enumerator = source.GetEnumerator();
```

Comment:
```suggestion
            using var enumerator = source.GetEnumerator();
```

Reply:
IEnumerator can not implicitly cast to System.IDisposable